draft-ietf-intarea-provisioning-domains.md

```diff
@@ -946,6 +954,13 @@ communicate with. In such scenarios, the host SHOULD check that the
 provided PvD ID, as well as the IP address that it resolves into, are
 part of the allowed whitelist.
 
+Network operators might want to restrict access to PvD Additional
+Information to only expose this to hosts that are connected to the local
+network, especially if the Additional Information would provide information
+about local network configuration to attackers. In such cases, operators
+SHOULD filter access to the Additional Information server, such as by
+whitelisting access from the addresses and prefixes that the router provides.
```

Comment:
Could also limit access to source address from the "prefixes" key in the JSON contents.

Both of these assume there's no proxy in use that's outside of these ranges, though...

Reply:
Fetching the additional information through a proxy may not be a desired case though?

Reply:
IMHO any use of proxies can only add complexity, ergo... =)

Maybe it's best left out.  But the idea of limiting the access to the PVD info to source addresses in the "prefixes" keyword (plus some SRE/admin ranges) seems to make operational sense to me.

Reply:
> Could also limit access to source address from the "prefixes" key in the JSON contents.
> 
> Both of these assume there's no proxy in use that's outside of these ranges, though...

BTW, this was the original intent of the original text. We can be more specific/clear indeed then using RECOMMEND rather than SHOULD.

Reply:
There is also a MITM attack if the PvD AI server serves the JSON file to an invalid address (the attack could use NPTv6 to victims pretending to use the expect PIO).